rgw/doc.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package rgw contains a set of wrappers around Ceph's RGW.
```

Comment:
We should probably be upfront and simply say that this is a placeholder.
Something like:
```suggestion
Package rgw contains sub-packages pertaining to the administration of RGW. It currently contains no functionally.
```
perhaps?